### PR TITLE
Moriya/settings l10n

### PR DESCRIPTION
**Pull request by moriya2004**
I added the settings folder
And I also added a location in the registration
eslint react

### CODE REVIEW

**Review by Shira-Abrams — APPROVED** [review]

File: client-side/src/components/profileComponents/addProfileComponent.jsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ import '../../styles/profilePageStyle.scss';
       }
       dispatch(addProfile(ProfileNew));
       setTimeout(() => navigate(0), 2000);
-      handleClose();
+      // handleClose();
```
Comment (moriya2004) [inline client-side/src/components/profileComponents/addProfileComponent.jsx:97]: this is not my component
It gives an error when running the project
This function does not exist in the component

File: client-side/src/components/profileComponents/profileActions.jsx [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
 import { Tooltip, Button } from '@mui/material';
 import { TOOLTIP_TEXTS, BUTTON_LABELS } from '../../constants/profileConstants.js';
-import '../../styles/profilePageStyle.scss';
 
- function ProfileActions({
-    handleSave = () => {}, 
-    handleClose = () => {}, 
-    handleDelete = () => {}
-}) {
+export default function ProfileActions({ handleSave, handleClose, handleDelete }) {
```
Comment (moriya2004) [inline client-side/src/components/profileComponents/profileActions.jsx:5]: i pull it from main
i didnt change it

File: client-side/src/App.jsx [hunk 3]
```diff
@@ -1,35 +1,24 @@
...
-    </Provider>
-   </SnackbarProvider>
+    <>
+      <SnackbarProvider maxSnack={3}>
+        <Provider store={store}>
+          <RouterProvider router={router} />
+          <Footer />
+        </Provider>
+      </SnackbarProvider>
+    </>
```
Comment (moriya2004) [inline client-side/src/App.jsx:21]: i pull it from main
i didnt change it

File: client-side/src/components/profileComponents/profileForm.jsx [hunk 4]
```diff
@@ -71,27 +70,3 @@ import { handleFieldChange } from '../../utils/profileUtil.js';
         </Box>
     );
 }
```
Comment (moriya2004) [inline client-side/src/components/profileComponents/profileForm.jsx:72]: i pull it from main
i didnt change it

File: client-side/src/components/settings/AccountTab.jsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const AccountTab = () => {
+    return (
+        <div>
+        </div>
+    )
+}
+export default AccountTab
+// TODO delete this component
```
Comment (moriya2004) [inline client-side/src/components/settings/AccountTab.jsx:10]: my friend will push her code in here

File: client-side/src/components/signUp/signUp.jsx [hunk 6]
```diff
@@ -29,32 +32,86 @@ const SignUpSchema = Yup.object().shape({
...
+      await createUser(userWithPreferences);
+      dispatch(addUser(userWithPreferences));
+      const loginResponse = await axios.post(`${apiUrl}/users/signIn`, user, {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+      const userData = loginResponse.data;
+      localStorage.setItem('userId', userData._id);
+      dispatch(setCurrentUser(userData));
```
Comment (moriya2004) [inline client-side/src/components/signUp/signUp.jsx:100]: i copy this func from login page

**Review by EA171 — APPROVED** [review]

**Review by e171271 — APPROVED** [review]

**Pull request merged**